my_game.py

```diff
@@ -233,7 +232,7 @@ class BonusUFO(arcade.Sprite):
     sound_fire = arcade.load_sound("sounds/laserRetro_001.ogg")
     sound_explosion = arcade.load_sound("sounds/explosionCrunch_000.ogg")
 
-    def __int__(self, shot_list, level=1, **kwargs):
+    def __int__(self, shot_list, level=1,target=Player ,**kwargs):
```

Comment:
The Player default value does not make sense, because it's a Class, not an instantiated object.